src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -765,8 +745,7 @@ def testImportHttpFile(self):
                     httpThread.join()
             finally:
                 http.server_close()
-       
-        @travis_test
```

Comment:
How come you don't want these to be Travis tests anymore?

Reply:
These tests are inherited by the AWS Jobstore Class which means that some of the AWS jobstore tests will only run on travis, but travis doesn't have `aws` so they're skipped there too.